README.md

```diff
@@ -77,6 +80,9 @@ For this reason we are unsetting _DOCKER_TLS_VERIFY_ for now.
   command prior the registration has taken place. To avoid this, uninstall the
   vbguest plugin or add the following to the _Vagrantfile_: `config.vbguest.
   auto_update = false`.
+* Causes of failure on Windows
```

Comment:
This is a good one. I had the same issue as well. Once the Vagrant session is interactive you need this setting set.